src/components/styles.js

```diff
@@ -0,0 +1,108 @@
+const ptrDark = `
+.__PREFIX__ptr {
+  pointer-events: none;
+  font-size: 0.85em;
+  font-weight: bold;
+  top: 0;
+  height: 0;
+  transition: height 0.3s, min-height 0.3s;
+  text-align: center;
+  width: 100%;
+  overflow: hidden;
+  display: flex;
+  align-items: flex-end;
+  align-content: stretch;
+
+  background: #303030;
+}
+
+.__PREFIX__box {
+  padding: 10px;
+  flex-basis: 100%;
+}
+
+.__PREFIX__pull {
+  transition: none;
+}
+
+.__PREFIX__text {
+  margin-top: .33em;
+  color: #ffffff;
+}
+
+.__PREFIX__icon {
+  color: #ffffff;
+  transition: transform .3s;
+}
```

Comment:
I'm not sure that mixing css styles in javascript var as a string is a good idea, but I can't find more efficient way to deal with better than this.